drivers/usb/udc/udc_common.h

```diff
@@ -61,21 +61,19 @@ struct udc_ep_config *udc_get_ep_cfg(const struct device *dev,
 /**
  * @brief Checks if the endpoint is busy
  *
- * @param[in] dev    Pointer to device struct of the driver instance
- * @param[in] ep     Endpoint address
+ * @param[in] ep_cfg Pointer to endpoint configuration
  *
  * @return true if endpoint is busy
  */
-bool udc_ep_is_busy(const struct device *dev, const uint8_t ep);
+bool udc_ep_is_busy(const struct udc_ep_config *const ep_cfg);
```

Comment:
What is the point of this `const`? Pointer parameters are provided by value.
```suggestion
bool udc_ep_is_busy(const struct udc_ep_config *ep_cfg);
```
Ditto on all other such cases in this file and `udc_common.c`

Reply:
It is common in device-next to have constant pointers. This const is to make sure that inside the function the pointer won't be modified (the "caller" value cannot be modified, regardless if the const is there or not, because as you pointed out the pointers are provided by value).